test/functional/rpc_fundrawtransaction.py

```diff
@@ -472,6 +488,8 @@ def run_test(self):
         connect_nodes_bi(self.nodes,1,2)
         connect_nodes_bi(self.nodes,0,2)
         connect_nodes_bi(self.nodes,0,3)
+        # Again lock the watchonly UTXO or nodes[0] may spend it
+        self.nodes[0].lockunspent(False, [{"txid": watchonly_txid, "vout": watchonly_vout}])
```

Comment:
Nit, add comment like "because restarting a node looses the locked unspents" (you may find better wording).

Reply:
Done!